content/docs/reference-events.md

```diff
@@ -256,33 +253,33 @@ string pointerType
 boolean isPrimary
 ```
 
-A note on cross-browser support:
+क्रॉस-ब्राउज़र समर्थन पर ध्यान दें:
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+पॉइंटर ईवेंट्स अभी तक हर ब्राउज़र में समर्थित नहीं हैं (इस लेख को लिखने के समय, समर्थित ब्राउज़र में शामिल हैं: Chrome, Firefox, Edge और Internet Explorer)। React जानबूझकर अन्य ब्राउज़रों के लिए पॉलीफिल समर्थन नहीं करता है क्योंकि एक मानक-अनुरूप पॉलीफिल `react-dom` के बंडल आकार में काफी वृद्धि करेगा।
```

Comment:
reorder to समर्थित ब्राउज़र में Chrome, Firefox, Edge और Internet Explorer शामिल हैं।

Reply:
second sentence is too complex

Reply:
समर्थन could be replaced by something else, rest is great :)